src/Routes/web.php

```diff
@@ -11,5 +11,6 @@
     ->middleware(['web', HandleInertiaRequests::class])
     ->group(function () {
         Route::get('/', [InertiaController::class, 'home']);
+        Route::get('/course/{id}/people', [InertiaController::class, 'people']);
```

Comment:
I think this one should be grouped with other course URLs, please review my PR https://github.com/uasoft-indonesia/badaso-lms-theme/pull/20 so we can group it later